.github/workflows/ui-test.yml

```
@@ -64,6 +64,7 @@ jobs:
 
       - name: Upgrade
         run: |
+         docker exec -t prestashop php modules/autoupgrade/bin/console backup:create admin-dev
          docker exec -t prestashop php modules/autoupgrade/bin/console update:start --channel=${{ matrix.UPGRADE_CHANNEL }} admin-dev
```

Actually can we move this in a new step?

```suggestion
      - name: Backup
        run: docker exec -t prestashop php modules/autoupgrade/bin/console backup:create admin-dev
      - name: Upgrade
        run: |
         docker exec -t prestashop php modules/autoupgrade/bin/console update:start --channel=${{ matrix.UPGRADE_CHANNEL }} admin-dev
```